### PR TITLE
Added Watchdog docs

### PR DESCRIPTION
**Pull request by ntimo**
This PR adds docs about the Watchdog

### CODE REVIEW

File: docs/u_e-watchdog.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+- Container failure and restart
+- Netfilter bans
+
+If you do not want to receive ban notification emails change `y` to `n`: 
+```
+# Notify about banned IP (includes whois lookup)
+WATCHDOG_NOTIFY_BAN=y
+```
+
+Please note that you need to create a SPF record for your MAILCOW_HOSTNAME. Otherwise the recipient mailserver could reject the email. Its very important that if you have a SPF record it has to be correct so please take the time and check that it is indeed valid.
```
Comment (fm) [inline docs/u_e-watchdog.md]: Change `Its` to `It's` as you are using it as a contraction for "it is"

File: docs/u_e-watchdog.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+Please note that if you organisation domain as a DMARC record you should disable DMARC for the MAILCOW_HOSTNAME domain by setting:
```
Comment (fm) [inline docs/u_e-watchdog.md]: `if you organisation domain as a` change to `if your organisation domain has a`

File: docs/u_e-watchdog.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+The watchdog is a container that will continues monitor your Mailcow installation, and restart containers that are unhealty.
```
Comment (fm) [inline docs/u_e-watchdog.md]: Change `unhealty` to `unhealthy`

File: docs/u_e-watchdog.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+
+By default the watchdog is disabled to enable it change `n` to `y`:
```
Comment (fm) [inline docs/u_e-watchdog.md]: Add a semi colon `;` after disabled

File: docs/u_e-watchdog.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+```
+
+You can also add your email address via:
+```
+# Send notifications by mail (no DKIM signature, sent from watchdog@MAILCOW_HOSTNAME)
+# Can by multiple rcpts, NO quotation marks
+#WATCHDOG_NOTIFY_EMAIL=a@example.com,b@example.com,c@example.com
+```
+
+Once you did that you will receive email notifications for the following type of events:
```
Comment (fm) [inline docs/u_e-watchdog.md]: Change `Once you did that` to `Once added,`

**Comment by milkmaker** [comment]
This pull request has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs.

**Comment by milkmaker** [comment]
This pull request has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs.

**Comment by milkmaker** [comment]
This pull request has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs.

**Comment by milkmaker** [comment]
This pull request has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs.

**Comment by milkmaker** [comment]
This pull request has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs.

**Comment by milkmaker** [comment]
This pull request has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs.

**Pull request closed without merging**